articles/openshift/oshift-how-view-proxy-logs.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to view proxy logs in your multinetwork OpenShift cluster | UKCloud Ltd
+description: Explains how to view the logs of the squid proxy in your cluster
+services: openshift
+author: Kieran O'Neill
+reviewer:
+lastreviewed: 
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to view proxy logs in your multinetwork OpenShift cluster
+toc_fullpath: How To/oshift-how-view-proxy-logs.md
+toc_mdlink: oshift-how-view-proxy-logs.md
+---
+
+# How to view proxy logs in your multinetwork OpenShift cluster
+
+## Overview
+
+UKCloud for OpenShift enables you to develop, deploy, and manage digital and container-based applications seamlessly across local physical or virtual environments, with full portability to and from UKCloud.
+
+This article explains how you can view the connection logs generated by the squid proxy in your OpenShift cluster.
+
+## Prerequisites
+
+Your OpenShift cluster must have access to multiple networks, be version 3.11 or later, and must have proxy logging enabled. Any OpenShift clusters deployed from November 2019 will have proxy logging enabled. If you have an older OpenShift cluster and would like proxy logging enabled then raise a request on the UKCloud Portal requesting this functionality.
+
+## Viewing logs
+
+In multinetwork 3.11 clusters we run a squid proxy on the controlplane loadbalancers in order to authenticate to Red Hats container registry 'registry.redhat.io'. This also allows you to hit internet endpoints from community network nodes providing you pass the correct variables to your pods and have whitelisted the endpoint. You may inadvertently be traversing the proxy where you do not want to or attempting to hit endpoints that are not whitelisted which will cause you to get 403 forbidden errors. 
```

Comment:
"authenticate" - would be better to say "access"
"Red Hats" should be "Red Hat's"

"This also allows you" should be "The proxy also allows you" because it isn't "accessing Red Hat's container registry that lets you hit internet endpoints.

Reply:
Changes made